src/backend/distributed/sql/udfs/citus_stat_activity/latest.sql

```diff
@@ -0,0 +1,13 @@
+DROP VIEW IF EXISTS pg_catalog.citus_stat_activity;
+
+CREATE VIEW citus.citus_stat_activity AS
+SELECT global_pid, nodeid, pg_stat_activity_from_all_nodes.* FROM
+run_command_on_all_nodes($$
```

Comment:
maybe we should use `parallel=true`? what is your take?

Reply:
I guess that is OK and it is also the default.